src/components/QuoteBubble/__tests__/BubbleButtons.test.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react';
 import { create } from 'react-test-renderer';
-import { cleanup } from '@testing-library/react';
 
 import BubbleButtons from '../BubbleButtons';
 
-afterEach(cleanup);
-
-jest.mock();
+jest.mock('../CopyButton', () => 'CopyButton');
+jest.mock('../DesktopShare', () => 'DesktopShare');
+jest.mock('../MobileShare', () => 'MobileShare');
 
 describe('COMPONENT - QuoteBubble BubbleButtons', () => {
-  it('renders correctly for Facebook variant', () => {
+  it('renders correctly if navigator share API is availible', () => {
+    global.navigator.share = jest.fn();
```

Comment:
Either we declare `navigator` here, or in `setupTests.js`. If you want to keep it here, please remove the declaration from setup file.

Reply:
It's just here for clarity.